### ISSUE TITLE
AttributeError in DropboxFile

### ISSUE THREAD

**Issue by MichaelPak** [jschneier/django-storages#620]
`Version 1.7.1`

Trying to get bytes from `DropboxFile`. It has property `file`:
```python
@property
def file(self):
    if not hasattr(self, '_file'):
        response = self._storage.client.files_download(self.name)
        self._file = SpooledTemporaryFile()
        copyfileobj(response, self._file)
        self._file.seek(0)
    return self._file
```
But `response` is a tuple of objects `Response` and` FileMetadata`, so I'm getting next error:
```
  File "/.../myapp/mailing.py", line 50, in distribute
    part.set_payload(offer_file.file.read())
  File "/.../python3.6/site-packages/django/core/files/utils.py", line 16, in <lambda>
    read = property(lambda self: self.file.read)
  File "/.../python3.6/site-packages/django/core/files/utils.py", line 16, in <lambda>
    read = property(lambda self: self.file.read)
  File "/.../python3.6/site-packages/storages/backends/dropbox.py", line 45, in file
    copyfileobj(response, self._file)
  File "/.../python3.6/shutil.py", line 79, in copyfileobj
    buf = fsrc.read(length)
AttributeError: 'tuple' object has no attribute 'read'
```

**Comment by sww314** [comment]
This is fixed in #402. However, that pull request is not ready to merge. You can try the patch and see if it works for you.

**Comment by MichaelPak** [comment]
I've installed `django-storages` from @danni's [fork](https://github.com/squareweave/django-storages) and it's work for me.

As far as I understand, the error appeared due to the update of `dropbox` package. Can you tell which version of `dropbox` I should use?

**Comment by jschneier** [comment]
Fixed by #724.
